CreateOwnAdventure

```diff
@@ -0,0 +1,12 @@
+Hello avid travelers. 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.